examples/test_examples.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function, division, absolute_import
+import six
```

Comment:
I'd rather not add a dependency for this.. we can embed `six.py` under `pyqtgraph/util`, though.
Presently I use `pyqtgraph/python2_3.py` for this purpose (as it predates my awareness of `six`).